pkgs/development/python-modules/base58/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pytest, pytestpep8
+, pytestflakes, pytestcov, pyhamcrest }:
+
+buildPythonPackage rec {
+  pname = "base58";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "keis";
+    repo = "base58";
+    rev = "871dbe14fe8e9675ef262a5e06fbf3874562a28c";
+    sha256 = "0f8isdpvbgw0sqn9bj7hk47y8akpvdl8sn6rkszla0xb92ywj0f6";
+  };
+
+  buildInputs = [ pytest pytestpep8 pytestflakes pytestcov pyhamcrest ];
+  checkPhase = ''
+    pytest --pep8 --flakes --cov=base58
```

Comment:
It would be better to remove the flags from this line and appropriate packages from `checkInputs`. They are not interesting for Nixpkgs.

Reply:
done